build.gradle.kts

```diff
@@ -72,7 +74,7 @@ tasks
         }
     }
 
-fun webdriver(module: String): String = "org.seleniumhq.selenium:$module:3.11.0"
+fun webdriver(module: String): String = "org.seleniumhq.selenium:$module:$seleniumVersion"
```

Comment:
Let's add a `### Fixed` change log entry.
Releasing a version with an empty change log would be confusing for consumers.

Reply:
Done